nekoyume/Assets/_Scripts/UI/Widget/Popup/NoticePopup.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using UnityEngine.UI;
+using UnityEngine;
+using Nekoyume.Game.Controller;
+using Nekoyume.Helper;
+
+namespace Nekoyume.UI
+{
+    public class NoticePopup : PopupWidget
+    {
+        [Serializable]
+        public class NoticeInfo
+        {
+            public string name;
+            public Sprite contentImage;
+            public string beginTime;
+            public string endTime;
+            public string pageUrlFormat;
+        }
+
+        [SerializeField] private Image contentImage;
```

Comment:
```suggestion
        [SerializeField]
        private Image contentImage;
```

This project has code convention about attribute.
after [attribute], please input newline.

Reply:
I Fixed It! Where can I check this project's convention?